controllers/mover/syncthing/syncthing_test.go

```diff
@@ -474,7 +477,62 @@ var _ = Describe("When an RS specifies Syncthing", func() {
 		})
 
 		Context("Syncthing API is being used properly", func() {
-			// TODO: test syncthing API
+			When("Syncthing server does not exist", func() {
+				It("Doesn't work", func() {
```

Comment:
This is a good start, though you should also test the `mover`'s `ensureIsConfigured()` and `mover.ensureStatusIsUpdated()` methods here to make sure that they're both erroring out correctly.